content/en/security/cloud_security_management/misconfigurations/compliance_rules.md

```diff
@@ -25,7 +25,11 @@ The compliance rules follow the same [conditional logic][2] as all Datadog Secur
 Cloud Security Misconfigurations uses the following rule types to validate the configuration of your cloud infrastructure:
 
 - [**Cloud configuration**][1]: These compliance rules analyze the configuration of resources within your cloud environment. For example, the [CloudFront distribution should be encrypted][3] rule assesses whether an Amazon CloudFront distribution enforces HTTPS to secure communications.
-- [**Infrastructure configuration**][5]: These checks evaluate containers and Kubernetes clusters using rules from CIS compliance benchmarks for Docker and Kubernetes, as well as Linux workloads against CIS host benchmarks for Ubuntu, Red Hat, and Amazon Linux.
+- [**Infrastructure configuration**][5]: These checks evaluate:
```

Comment:
```suggestion
- [**Infrastructure configuration**][5] checks evaluate:
```

Reply:
Optional suggestion here that I find a bit easier to visually parse. Feel free to ignore this if you have a reason for the double colon here!